tests/conftest.py

```diff
@@ -87,6 +92,13 @@ def multi_agent_observation(woodworking_ma_combined_domain: Domain,
         WOODWORKING_COMBINED_TRAJECTORY_PATH, executing_agents=WOODWORKING_AGENT_NAMES)
 
 
+@fixture()
+def multi_agent_observation2(woodworking_ma_combined_domain: Domain,
```

Comment:
I don't like this.... I know why you did it but I would rather identify the bug. For now this is ok, just comment this with a TODO with the issue or something so that we'd know to return and delete this.